style/rails/README.md

```diff
@@ -8,7 +8,8 @@ Rails
 * Name time columns (referring to a time of day with no date) with `_time`
   suffixes.
 * Name initializers for their gem name.
-* Order ActiveRecord associations alphabetically by attribute name.
+* Order ActiveRecord associations alphabetically by association type, then
+  attribute name. [Example][order-associations]
```

Comment:
I'm not opposed to that idea, thought that would be inconsistent with how we've been providing examples so far. I'm okay with it if we decided to do examples in this manner going forth.